### PR TITLE
Prevent TNT from blowing up Map Tools nodes

### LINKED ISSUE #21: "Unbreakable" nodes are TNT-able
This is an easy fix (add `on_blast = function() end` to the definitions), but I wanted to check to see if this was perhaps the intended behaviour. I can create a PR if someone else thinks it's a problem.

### PR DESCRIPTION
**Pull request by fluxionary**
Fixes #21 

Also did a little refactoring while I was in there, I hope that was appropriate.

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
LGTM. 👍

File: default_nodes.lua [hunk 1]
```diff
@@ -9,287 +9,190 @@ local S = maptools.S
...
+	def.stack_max = 10000
+	def.drop = ""
+	if def.groups then
+		def.groups.unbreakable = 1
+		def.groups.not_in_creative_inventory = maptools.creative
+	else
+		def.groups = {unbreakable = 1, not_in_creative_inventory = maptools.creative}
+	end
+	def.on_drop = maptools.drop_msg
+	def.on_blast = function() end
```
Comment (Calinou) [inline default_nodes.lua:25]:
```suggestion
	-- Prevent Map Tools nodes from being exploded by TNT.
	def.on_blast = function() end
```

File: default_nodes.lua [hunk 2]
```diff
@@ -9,287 +9,190 @@ local S = maptools.S
 
 maptools.creative = maptools.config["hide_from_creative_inventory"]
 
-minetest.register_node("maptools:stone", {
+local function register_node(name, def)
+	def.range = 12
```
Comment (Calinou) [inline default_nodes.lua:14]:
```suggestion
	-- Increase the interaction range when holding Map Tools nodes to make building easier.
	def.range = 12
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**